src/server.js

```diff
@@ -164,8 +164,9 @@ const createServer = (config, db, log, Database) => {
         log(`authorization server returned [${authResponse.status}] response`);
         log('token validated by authorization server');
 
-        const validToken = await authResponse.json();
-        const isValid = validToken.active === true;
+        const authResponseToken = await authResponse.json();
+        log('auth response token', authResponseToken);
```

Comment:
possibly dont want to log anything sensitive like an access token.

Reply:
Mmmmhmmm yeah in my development fugue I sorta figured that the expiry + the people reading it would make it okay, but I think you're right: best err on the side of caution.